ide-support/lsp/lsp-server/src/main/java/io/helidon/lsp/server/management/MavenSupport.java

```diff
@@ -145,13 +159,14 @@ public Set<Dependency> dependencies(final String pomPath, int timeout) {
 
     private static String getMavenVersion() {
         final Properties properties = new Properties();
-        final String corePomProperties = "META-INF/maven/io.helidon.ide-support.lsp/io.helidon.lsp.server/pom.properties";
+        final String corePomProperties = "META-INF/maven/io.helidon.ide-support.lsp/helidon-lsp-server/pom.properties";
 
         try (InputStream in = MavenSupport.class.getClassLoader().getResourceAsStream(corePomProperties)) {
             if (in != null) {
                 properties.load(in);
             }
         } catch (IOException ioe) {
+            LanguageClientLogUtil.logMessage("", ioe);
```

Comment:
What does it mean to return an empty string for the version. Is this a valid use-case that we can't get the resource ? Shouldn't it be just an error instead ? (i.e. throw an exception)

Reply:
In this case it will try to use the latest version of the maven plugin